client/src/modules/list/components/Items/ListArchivedItem/ListArchivedItem.scss

```diff
@@ -29,8 +29,16 @@
   }
 
   &__name {
-    font-weight: 500;
     color: $text-dark;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 1.2;
+    margin-bottom: 4px;
+    max-width: 210px;
```

Comment:
Please also add `title={originalText}` attribute so the full name is visible on hover (uses native tooltip).